cpp/src/arrow/compute/exec/aggregate.h

```diff
@@ -37,6 +37,7 @@ namespace internal {
 /// and use an aggregate node.
```

Comment:
I think this comment is not right because `GroupBy` is also used by `pyarrow._compute._group_by`, which is also exposed in `pyarrow.compute`.